example-get-started/generate.sh

```diff
@@ -63,7 +63,16 @@ git tag -a "1-dvc-init" -m "DVC initialized."
 mkdir data
 dvc get https://github.com/iterative/dataset-registry \
   get-started/data.xml -o data/data.xml
-dvc add data/data.xml --desc "Initial XML StackOverflow dataset (raw data)"
+python <<EOF
+from dvc.repo import Repo
```

Comment:
can we also use `log_artifact`?

Reply:
We can, but I don't think it's logical (it's either a separate python script or modifying `prepare.py`, which doesn't make sense to me). Rather I can simply echo few lines from bash if that's better.

Reply:
wdyt?